auraed/src/runtime/mod.rs

```diff
@@ -292,41 +295,15 @@ impl cell_service_server::CellService for CellService {
     }
 }
 
-// Here is where we define the "default" cgroup parameters for Aurae cells
-fn create_cgroup(id: &str, cpu_shares: u64) -> Result<Cgroup, Error> {
-    let hierarchy = hierarchy();
-    let cgroup: Cgroup = CgroupBuilder::new(id)
-        .cpu()
-        .shares(cpu_shares) // Use x% of the CPU relative to other cgroups
-        .done()
-        .build(hierarchy);
-    Ok(cgroup)
-}
-
-fn remove_cgroup(id: &str) -> Result<(), Error> {
-    // TODO: create a cgroup_table mapping cell name to cgroup and do this instead
-    //if let Err(err) = cgroup.delete() {
-    //    return Err(Error::from(err))
-    //}
-    //Ok(())
-
-    // The 'rmdir' command line tool from GNU coreutils calls the rmdir(2)
-    // system call directly using the 'unistd.h' header file.
-
-    // https://docs.rs/libc/latest/libc/fn.rmdir.html
-    let path = std::ffi::CString::new(format!("/sys/fs/cgroup/{}", id))?;
-    let ret = unsafe { libc::rmdir(path.as_ptr()) };
-    if ret < 0 {
-        let error = io::Error::last_os_error();
-        error!("Failed to remove cgroup ({})", error);
-        Err(Error::from(error))
-    } else {
-        Ok(())
-    }
-}
-
 fn hierarchy() -> Box<dyn Hierarchy> {
-    hierarchies::auto() // v1/v2 cgroup switch automatically
+    // Auraed will assume the V2 cgroup hierarchy by default.
+    // For now we do not change this, albeit in theory we could
+    // likely create backwards compatability for V1 hierarchy.
+    //
+    // For now, we simply... don't.
+    // hierarchies::auto() // Uncomment to auto detect Cgroup hierarchy
+    // hierarchies::V2
+    Box::new(hierarchies::V2::new())
```

Comment:
do we need this method? can we not just do this inline?

Reply:
I would prefer to have a method that everything calls, such that we can easily change it and add logic in the future. It seems to be safer than just hard coding everything.